unfurl/repo.py

```diff
@@ -60,9 +59,8 @@ def split_git_url(url):
 
 class _ProgressPrinter(git.RemoteProgress):
     def update(self, op_code, cur_count, max_count=None, message=""):
-        # print update to stdout but only if logging is INFO or more verbose
-        if message and get_log_level() <= logging.INFO:
-            print("fetching from %s, received: %s " % (self.gitUrl, message))
```

Comment:
I didn't know the reason why we are using print instead of logging.info. I thought it's a mistake. I reverted my changes and added a comment with explanation.